types/label.d.ts

```diff
@@ -106,6 +106,10 @@ export interface LabelOptions extends ContainedLabelOptions {
    * @deprecated
    */
   yPadding?: Scriptable<number, PartialEventContext>,
+  shadowColor: Scriptable<Color, PartialEventContext>,
+  shadowBlur: Scriptable<number, PartialEventContext>,
+  shadowOffsetX: Scriptable<number, PartialEventContext>,
+  shadowOffsetY: Scriptable<number, PartialEventContext>
```

Comment:
Missing the `?` behind the property name to make them optional

Reply:
:+1: done!